GameData/RealismOverhaul/TestFlightRnD.cfg

```diff
@@ -1,26 +1,48 @@
-// Set the global TF RnD settings
+//  ==================================================
+//  Set the global TF RnD settings.
+//  ==================================================
+
 TFRNDSETTINGS
 {
     updateFrequency = 86400
+
     TEAM
     {
         points = 10.0
         costFactor = 1
     }
+
     TEAM
     {
         points = 12.5
         costFactor = 1.5
     }
+
     TEAM
     {
         points = 15.0
         costFactor = 2
     }
 }
 
-// After pass: fix up RnD
-@PART[*]:HAS[@MODULE[TestFlightCore]]:AFTER[zTestFlight]
+
+//  ==================================================
+//  Before pass: check if any parts are missing an
+//  entry cost field. If so, add one and tag them.
+//  ==================================================
+
+@PART[*]:HAS[~entryCost[]]:AFTER[RealismOverhaul]
```

Comment:
Needs to be `:AFTER[RealismOverhaulEngines]` or similar.

otherwise the description gets overwritten later.